JS-Algorithms/challenges.js

```diff
@@ -117,9 +172,17 @@ Invoking "fibIndex(3)" should return "12".
 Because the 12th index in the Fibonacci sequence is 144, and 144 has three digits
 ***** */
 
-const fibIndex = (n) => {
-  // YOUR CODE HERE...
-};
+function fibIndex(n) {
+  let fib = [0n, 1n];
+  let index = 0;
+
+  for (let i = 2; fib[fib.length - 1].toString().length != n; i++) {
+    fib.push(fib[i - 1] + fib[i - 2]);
+    index++;
+  }
+
+  return index + 1;
```

Comment:
Great job with this function, there are some improvements that can be done.
* Using a number instead of an array for the calculation to avoid large storage usage